root/usr/bin/discovery-debug

```diff
@@ -28,6 +28,9 @@ ip addr show
 echo "* ROUTES *"
 ip route
 
+echo "* NEIGHBOURS *"
```

Comment:
Hmmm indeed, but what for? I'd rather not report things we don't process. I added this because I sometimes mis-start a VM with multiple NICs on the same network.